fellow-submissions/devonte-duncan/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang=en>
+<head>
+<title>Devy Flex</title>
+<meta charset=UTF-8>
+<link rel=stylesheet href=styles.css type=text/css />
+</head>
+<body>
+    <header>
+        <h1>Devy Creation Corps.</h1>
+        <ul>
+          <li><a href="#">Home</a></li>
+          <li><a href="#">About Us</a></li>
+          <li><a href="#">Our Clients</a></li>
+          <li><a href="#">Contact</a></li>
+          <li><a href="#">Team</a></li>
+        </ul>
+    </header>
+    <main>
+        <section>
+            <div>
+                
+            </div>
+        </section>
+        
+        <article>
```

Comment:
You usually won't see an aside wrapped in an article. You could have created them as siblings like...
```html
<main>
  <section>
    ...
  </section>
  <aside>
   ...
  </aside>
  <article>
    ...
  </article>
.
.
.
```